.circleci/config.yml

```diff
@@ -2,6 +2,14 @@ version: 2
 
 jobs:
   assemble:
+    environment:
+      GRADLE_ARGS: >-
```

Comment:
Fancy.

Reply:
[YAML is simple](https://yaml-multiline.info/) 🙃